src/leiningen/checkout/ln.clj

```diff
@@ -67,12 +67,17 @@
   (println "lein checkout enable; lein checkout " (str \" pattern \"))
   (println "# instead!"))
 
+(defn extract-vals
+  "Creates a function will will return a vector of values by looking up the keys ks in map m"
+  [ks]
+  (fn [m] (map m ks)))
+
 (defn ln
   "[pattern]: Link project(s) into checkouts. If PATTERN is specified, link all projects matching `.*PATTERN.*`."
   [{:keys [name dependencies] {:keys [search-roots]} :checkout :as project} & [pattern]]
-  (let [dependency-names                        (into #{} (map (comp :artifact-id project/dependency-map) dependencies))
+  (let [dependency-coords                       (into #{} (map (comp (extract-vals [:group-id :artifact-id]) project/dependency-map) dependencies))
         {:keys [projects search-roots]}         (apply checkout-candidates project search-roots)
-        candidates-for-checkout                 (filter (comp dependency-names :name) projects)
+        candidates-for-checkout                 (filter (comp dependency-coords (extract-vals [:group :name])) projects)
```

Comment:
Thanks for catching that. I love `juxt` and I was just using it elsewhere last week, but I didn't make the connection between it and the function I had added here. I've removed my function accordingly.